### DISCUSSION TITLE
Add an Upload Status Bar

### DISCUSSION THREAD

**Discussion by BLACK4585** [diced/zipline#487] [Ideas]
It would be great to see the current status of file uploads, not only the three dots. This is useful for larger files.

**Comment by diced** [comment]
There is a progress bar that shows up after you click upload..?

**Comment by BLACK4585** [comment]
![grafik](https://github.com/diced/zipline/assets/71935902/5923a5ab-d50a-4e52-ae5e-a1563668e10d)
It only shows me only this little message, right?

**Reply by diced** [reply]
There should be a progress bar under the buttons
![image](https://github.com/diced/zipline/assets/35403473/15b6d35a-c847-4982-a198-0944a5edb564)

**Comment by BLACK4585** [comment]
Hmmm weird..
![grafik](https://github.com/diced/zipline/assets/71935902/3d68c592-92ea-407a-88f3-7a2b3f74a7dd)
